tfs/src/main/java/hudson/plugins/tfs/rm/ReleaseManagementHttpClient.java

```diff
@@ -119,6 +162,36 @@ private String ExecuteGetMethod(String url) throws ReleaseManagementException
         
         return response;
     }
+
+    private InputStream ExecuteGetMethodAsStream(String url) throws ReleaseManagementException
+    {
+        GetMethod getMethod = new GetMethod(url);
+        getMethod.addRequestHeader("Authorization", this.basicAuth);
+        InputStream response;
+        try
+        {
+            int status = this.httpClient.executeMethod(getMethod);
+            response = getMethod.getResponseBodyAsStream();
+            if(status >= 300)
+            {
+                throw new ReleaseManagementException("Error occurred.%nStatus: " + status + "%nResponse: " + response + "%n");
+            }
+        }
+        catch(HttpException ex)
+        {
+            throw new ReleaseManagementException(ex);
+        }
+        catch(IOException ex)
```

Comment:
we are already dealing with the generic Exception. Do we need to catch the HttpException and IOException separately?

Reply:
I don't think it's useful to catch different types of exceptions but throw the same type in the end. However, I just follow the previous functions like `ExecuteGetMethod` or `ExecutePostmethod`.  So, do we need to also change these functions to just catch generic Exception.

Reply:
I am not sure how we ended up having that code to catch different exception but finally throw the same exception. Please fix  your method just to handle generic exception. Get and Post method I will fix it later.